roles/openshift_metrics/templates/hawkular_truncation_cron.j2

```diff
@@ -0,0 +1,34 @@
+---
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: hawkular-truncation-cron
+  labels:
+    metrics-infra: hawkular-metrics
+    name: hawkular-truncation-cron
+spec:
+  schedule: "{{ openshift_metrics_trunation_cron_schedule }}"
+  concurrencyPolicy: Forbid
+  successfulJobsHistoryLimit: 5
+  failedJobsHistoryLimit: 5
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          concurrencyPolicy: Forbid
+          replicas: 1
+          strategy:
+            type: Recreate
+          labels:
+              metrics-infra: hawkular-metrics
+          terminationGracePeriod: 800
+          serviceAccountName: hawkular-truncation-cron
+          restartPolicy: OnFailure
+          containers:
+          - name: "hawkular-truncation-cron"
+            image: {{ openshift_metrics_trunation_cron_image }}
+            env:
+            - name: METRICS_NAMESPACE
+              value: "{{openshift_metrics_project}}"
+            - name: PARTITION_THRESHOLD
+              value: "1048576"
```

Comment:
here you have a different PARTITION_THRESHOLD value than in the Dockerfile https://github.com/openshift/origin-metrics/pull/434/files#diff-6f9d5e4505707e46e89b0d0b81b00d01R29, also this might better be configurable?